celery/concurrency/asynpool.py

```diff
@@ -775,9 +775,6 @@ def _put_back(job, _time=time.time):
         def on_poll_start():
             # Determine which io descriptors are not busy
             inactive = diff(active_writes)
-            logger.debug(
```

Comment:
should we must remove this?

Reply:
Not must, but this log is extremely high frequency.
When I open debug level, I almost only see this log.
If this log is important, and should not delete, please let me know.

Reply:
I think this could be reverted as not related. we can handle that in another issue.

Reply:
Ok.